src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -0,0 +1,131 @@
+package bio.terra.service.snapshotbuilder;
+
+import bio.terra.model.SnapshotBuilderConcept;
+import bio.terra.model.SnapshotBuilderDomainOption;
+import bio.terra.model.SnapshotBuilderFeatureValueGroup;
+import bio.terra.model.SnapshotBuilderListOption;
+import bio.terra.model.SnapshotBuilderPrepackagedConceptSets;
+import bio.terra.model.SnapshotBuilderProgramDataOption;
+import bio.terra.model.SnapshotBuilderSettings;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SnapshotBuilderService {
+
+  @Autowired
+  public SnapshotBuilderService() {}
+
+  public SnapshotBuilderSettings getSnapshotBuilderSettings(UUID datasetId) {
```

Comment:
Could you please add a docstring to this method that talks about the usage of dummy data, since that's not something we've seen up to this point in TDR?

I don't think unit tests are strictly necessary.